testproject/Assets/Chick/Scripts/ChickController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MLAPI;
+using UnityEngine;
+
+[RequireComponent(typeof(Animator))]
+public class ChickController : NetworkedBehaviour
+{
+    private Animator m_Animator;
+
+    private void Awake()
+    {
+        m_Animator = GetComponent<Animator>();
+    }
+
+    private void Update()
+    {
+        if (IsOwner)
+        {
+            m_Animator.SetBool("Eat", Input.GetKey(KeyCode.C));
+        }
```

Comment:
Hold `C` to play `Eat` animation across the network! 😛

Reply:
Nice!